src/lib/stitching/exchange/stitching.ts

```diff
@@ -230,16 +233,18 @@ export const exchangeStitchingEnvironment = ({
           fragment: `fragment CommerceLineItemArtwork on CommerceLineItem { artworkVersionId }`,
           resolve: (parent, _args, context, info) => {
             const id = parent.artworkVersionId
-            return info.mergeInfo.delegateToSchema({
+            const globalID = toGlobalId("ArtworkVersion", id)
+            return delegateToSchema({
               schema: localSchema,
               operation: "query",
-              fieldName: "artworkVersion",
+              fieldName: "node",
               args: {
-                id,
+                id: globalID,
               },
               context,
               info,
               transforms: exchangeSchema.transforms,
+              returnType: ArtworkVersionType,
```

Comment:
@mzikherman this `returnType` field (only available on the new `delegateToSchema` function) is what automatically narrows the type down from `Node` to `ArtworkVersion`. This way, you don't have to do the `... on ArtworkVersion` verbosity in the query.

Reply:
oo awesome! So this plus the `graphql-tools` bump? That's sweet.